src/components/Grid.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import instruments from "../utilities/instruments";
+
+export default ({ grid, playAudio, changeSound }) => {
+  console.log(grid, "rendered");
+  const [instrument, setInstrument] = useState("none");
+  const options = {
+    none: null,
+    kick: instruments.kick,
+    hiHat: instruments.hiHat,
+    snare: instruments.snare,
+    tom: instruments.tom,
+    floorTom: instruments.floorTom
+  };
+  const handleChange = e => {
+    e.preventDefault();
+    setInstrument(e.target.value);
+  };
+  return (
+    <div style={{ backgroundColor: "black", height: "100vh" }}>
+      <table id="button-grid">
+        <tbody>
+          {grid.map((row, rowId) => {
+            return (
+              <tr key={rowId}>
+                {row.map((cell, cellId) => {
+                  return (
+                    <td
+                      key={`row${rowId}cell${cellId}`}
+                      style={{
+                        backgroundColor: `${
+                          cell.instrument ? "orange" : "grey"
+                        }`
+                      }}
+                      onClick={() =>
+                        changeSound(rowId, cellId, options[instrument])
+                      }
+                    ></td>
+                  );
+                })}
+              </tr>
+            );
+          })}
+        </tbody>
+      </table>
+      <button onClick={playAudio}>PLAY AUDIO</button>
+      <select onChange={handleChange} value={instrument}>
```

Comment:
This is really cool!  I'm sure you were pressed for time, but if you need to refactor a bit I think you could put these drum values into an array, and then make a function that maps over that array and for each item, return an option element with that item as a value.  Then you can call that function in the render. It will save you some lines!